msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignUpTest.kt

```diff
@@ -48,6 +48,29 @@ class SignUpTest : NativeAuthPublicClientApplicationAbstractTest() {
         Dispatchers.setMain(testDispatcher)
     }
 
+    private fun <T> retryOperation(
+        maxRetries: Int = 3,
+        onFailure: () -> Unit = { Assert.fail() },
+        block: () -> T
```

Comment:
great idea!
let's rename this var to something else? This to me sounds like "blocking", as in the test is being blocked or something.

Reply:
How about renaming it as authFlow()?